hw/ip/keymgr/data/keymgr_testplan.hjson

```diff
@@ -7,6 +7,7 @@
                      "hw/dv/tools/dvsim/testplans/intr_test_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/alert_test_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/tl_device_access_types_testplan.hjson",
+                     "hw/dv/tools/dvsim/testplans/shadow_reg_errors_testplan.hjson",
```

Comment:
Should we do an audit of all IPs to ensure all IPs that have shadow CSRs are importing this testplan (or maybe just publicize in an email).

Reply:
i will send an email once this PR is in. Thanks for the reminder.